src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionGridStyles.kt

```diff
@@ -23,6 +25,50 @@ class CollectionGridStyles: Stylesheet() {
 
         contentLoadingProgress {
             progressColor = AppTheme.colors.appRed
+            backgroundColor += Color.WHITE
+        }
+
+        tabHeaderBackground {
+            backgroundColor += Color.TRANSPARENT
+        }
+
+        val tabWidthCenter = mixin {
+            alignment = Pos.CENTER
+            prefWidth = 140.px // Must specify width for the label as well or center won't work
+        }
+
+        tab {
+            +tabWidthCenter
+            backgroundRadius += box(0.px, 0.px, 0.px, 0.px) // Remove beveled edges
+//            padding = box(0.px, 0.px, 0.px, 5.px) // Doesn't work
```

Comment:
I haven't been able to figure out how to get more space between the tabs.

Reply:
can we not put all of this tab styling inside a stylesheet? it looks out of place for it to be here in the tab.

Reply:
Nathan and I were talking about leaving it here yesterday, but now I think it might make sense to put the default tab styling in somewhere like MainScreenStyles because we will also have tabs on the takes page. I would probably keep Scripture, tN, etc. styling here since those tabs will only show up on the collections (chapter) page

Reply:
If it's not here I think it would make more sense to have the styling not in MainScreen but more AppStyles. I think it would get confusing because you don't actually see the tabs in the MainScreen but rather in other fragments that get docked in the workspace. since the mainscreen doesn't actually own those fragments it might make more sense for it be in AppStyles and have it in there as a general app wide default style.